libreantd

```diff
@@ -0,0 +1,31 @@
+start_libreant_daemon(){
+    if [ -f /libreant/ve/bin/libreant ];
+    then
+        /libreant/ve/bin/libreant --fsdb-path /fsdb-store &
```

Comment:
can't we just inline this command `/libreant/ve/bin/libreant --fsdb-path /fsdb-store & ` (which by the way doesn't create a daemon) in `Dockerfile` `RUN` section?
I don't see when/why stop will ever be called

Reply:
In a Dockerfile each ```RUN``` is a separate container. The changes made by each ```RUN``` are saved in a temporary container during the build and, if the build is successful, such temporary containers are then discarded. If you invoke a
```bash
RUN /libreant/ve/bin/libreant --fsdb-path /fsdb-store &
```
the following RUN will not have any libreant service up. This is why I wrote the _daemon_ and put in the ```ENTRYPOINT``` the ```service libreantd start```. 
I'd be more comfortable with a systemd unit but AFAIK it (still) does not work ([^](https://github.com/systemd/systemd/issues/1224))

Reply:
oh, I just meant `.ENTRYPOINT`
`ENTRYPOINT /libreant/ve/bin/libreant --fsdb-path /fsdb-store & && bash -i `

Reply:
Yes, I think we could do it but I liked the idea of having a `service` to `start|stop|restart` libreant 😄